docs/advanced_topics/synonyms.md

```diff
@@ -0,0 +1,348 @@
+# How to… use synonyms with Elasticsearch to improve search results
+
+Users might not always use the same terms as you to refer to something. An example might be 'waste', 'refuse', 'garbage', 'trash' and 'rubbish'. If you configure these as synonyms via the search engine, then you will not need to tag pages with every possible synonym, and a search query `Page.objects.search("gift")` could always return results including a page called "Donate to our Charity".
+
+```eval_rst note:: Stemming, and language-specific configuration
```

Comment:
This section is not formatting correctly in https://wagtail--6798.org.readthedocs.build/en/6798/advanced_topics/synonyms.html. I don't know what the correct syntax should be.

Reply:
@nimasmi 

```
```eval_rst
.. note:: Stemming, and language-specific configuration
```

should hopefully do it

Reply:
I suspect the contents of the note need to be indented one level. Failing that, try the long-hand eval_rst:

    ```eval_rst
    .. note:: Stemming, and language-specific configuration
        Synonym searching […]
    ```